blog.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ADOPTADOG</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
+  </head>
+
+  <body>
+      <header>
+        <div class='float-right'>
+          <img src="images/cart-icon.svg" alt="Shopping cart icon." width="30">
+          $1234.56<br>
+          <button>Checkout</button>
+        </div>
+        <img src="images/adoptadog-logo.svg" alt="Profile of a brown dog, staring at something." width="175">
```

Comment:
Please take out ALL of your css in your HTML. It can all be handled via the CSS file